src/python/test/test_wasserstein_with_tensors.py

```diff
@@ -0,0 +1,25 @@
+""" This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+    Author(s):       Mathieu Carriere
+
+    Copyright (C) 2020 Inria
+
+    Modification(s):
+      - YYYY/MM Author: Description of the modification
+"""
+
+from gudhi.wasserstein import wasserstein_distance as pot
+import numpy as np
+
+def test_wasserstein_distance_grad_tensorflow():
+    import tensorflow as tf
```

Comment:
Might as well import early with gudhi and numpy, if the whole file requires tensorflow.

Reply:
imports are now done at global level